### PR TITLE
[UID-159] Additional RTR debugging and configuration options

### PR DESCRIPTION
**Pull request by ncovercash**
https://github.com/folio-org/stripes-core/pull/1535 must be merged before this PR can pass

### CODE REVIEW

File: src/settings/RefreshTokenRotation.js [hunk 1]
```diff
@@ -1,70 +1,94 @@
...
   const [isLoading, setIsLoading] = useState(true);
   const [tokenExpiration, setTokenExpiration] = useState({});
 
   useEffect(() => {
-    getTokenExpiry()
-      .then(te => {
-        setTokenExpiration(te ?? { atExpires: -1, rtExpires: -1 });
-        setIsLoading(false);
-      });
+    setIsLoading(true);
```
Comment (ncovercash) [inline src/settings/RefreshTokenRotation.js:20]: this is goofy, but eslint got mad and insisted it was always false

**Review by mweaver-ebsco — APPROVED** [review]

File: src/settings/RefreshTokenRotation.js [hunk 2]
```diff
@@ -75,8 +99,55 @@ const RefreshTokenRotation = () => {
...
         <div>
-          <Button onClick={invalidateAT}><FormattedMessage id="ui-developer.rtr.invalidateAT" /></Button>
-          <Button onClick={invalidateRT}><FormattedMessage id="ui-developer.rtr.invalidateRT" /></Button>
+          <Button onClick={invalidateAT}>
+            <FormattedMessage id="ui-developer.rtr.invalidateAT" />
+          </Button>
+          <Button onClick={invalidateRT}>
+            <FormattedMessage id="ui-developer.rtr.invalidateRT" />
+          </Button>
+
```
Comment (zburke) [inline src/settings/RefreshTokenRotation.js:108]: Just purge this. It was effective when rotation was tied to API requests but it's no longer as relevant with rotation happening indepdently via a timer.

File: src/settings/RefreshTokenRotation.js [hunk 3]
```diff
@@ -1,70 +1,94 @@
...
-
-import {
-  Button,
-  LoadingPane,
-  Pane,
-  PaneHeader,
-} from '@folio/stripes/components';
+import { getTokenExpiry, setTokenExpiry } from '@folio/stripes/core';
+import { Button, LoadingPane, Pane, PaneHeader, TextField } from '@folio/stripes/components';
+import { RTR_FORCE_REFRESH_EVENT } from '../../../stripes-core/src/components/Root/constants';
```
[the remark this answers is not in the document]
Comment (ncovercash) [reply src/settings/RefreshTokenRotation.js:9]: Oops, yep - my IDE was trying to be clever with my workspace. Good catch!

**Comment by ncovercash** [comment]
@zburke should be fixed — also added the `fixedLengthSessionWarningTTL` from your eureka stripes-core merge!

**Pull request merged**